src/panel/direction/DirectionInput.jsx

```diff
@@ -16,34 +17,36 @@ class DirectionInput extends React.Component {
     inputRef: PropTypes.object.isRequired,
   }
 
-  componentDidMount() {
-    this.suggest = new Suggest({
-      tagSelector: `#itinerary_input_${this.props.pointType}`,
-      onSelect: this.selectItem,
-      withGeoloc: true,
-    });
+  state = {
+    mounted: false,
+    readOnly: false,
   }
 
-  componentWillUnmount() {
-    if (this.suggest) {
-      this.suggest.destroy();
-    }
+  componentDidMount() {
+    this.setState({
+      mounted: true,
+    });
   }
 
   onChange = event => {
     const value = event.target.value;
     this.props.onChangePoint(value, null);
   }
 
-  onKeyPress = event => {
+  onKeyPress = async event => {
     if (event.key === 'Enter' && this.props.value !== '') {
-      this.suggest.onSubmit();
+      const items = await fetchSuggests(this.props.value);
```

Comment:
I know it's related to how the Enter key behavior, but it doesn't seem right to have to do that here, whereas it's already done in the `Suggest` itself :/
And there is probably a bug also, as `fetchSuggests` include categories, which shouldn't be the case in the direction panel.

How can we solve that? Maybe managing the Enter key from outside wasn't the good way after all.

Reply:
withCategories is now set to false by default, thanks.

About the Enter key, after discussion we opted to let this PR as this, and think about another solution to implement later. I'm open to any idea.